### PR TITLE
Removed ImageryMaterialProperty.alpha

### LINKED ISSUE #3785: Remove ImageryMaterialProperty.alpha
I just noticed this in CHANGES.md:

> Deprecated `ImageryMaterialProperty.alpha`.  It will be removed in 1.21.  Use `ImageryMaterialProperty.color.alpha` instead.

### PR DESCRIPTION
**Pull request by TomPed**
Fixes #3785

Just to double check, I need to keep `this._alpha` correct?

### CODE REVIEW

**Comment by hpinkos** [comment]
@TomPed, no get rid of `_alpha` also.

You also need to remove alpha from [CzmlDataSource](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/DataSources/CzmlDataSource.js#L807-L810)

**Comment by TomPed** [comment]
Oh I see so this line can be removed too `color.alpha = Property.getValueOrDefault(this._alpha, time, defaultAlpha);` because `color = Color.clone(defaultColor, result.color);` takes care of the alpha for the color?

**Comment by TomPed** [comment]
Updated.

**Comment by TomPed** [comment]
Updated.

File: CHANGES.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Change Log
 ### 1.21 - 2016-05-02
 
 * Breaking changes
-  *
+  * Removed `ImageryMaterialProperty.alpha`.  It will be removed in 1.21.  Use `ImageryMaterialProperty.color.alpha` instead.
```
Comment (hpinkos) [inline CHANGES.md]: Delete `It will be removed in 1.21.`

File: Source/DataSources/ImageMaterialProperty.js [hunk 2]
```diff
@@ -155,7 +136,6 @@ define([
         var color = Property.getValueOrUndefined(this._color, time, result.color);
         if (!defined(color)) {
```
Comment (hpinkos) [inline Source/DataSources/ImageMaterialProperty.js]: Replace this if block with

`result.color = Property.getValueOrClonedDefault(this._color, time, defaultColor, result.color);`

Comment (TomPed) [inline Source/DataSources/ImageMaterialProperty.js]: When I do that I get this:

```javascript
result.color = Property.getValueOrClonedDefault(this._color, time, defaultColor, result.color);
result.color = color;
```
It seems odd that I am setting `result.color` twice in a row. Should I remove one of the statements?

Comment (hpinkos) [inline Source/DataSources/ImageMaterialProperty.js]: yeah, remove the second statement.    And remove the `var color...` line above the if statement as well.  We just needed that logic to make alpha still work for this past release.  Now that we're removing alpha, it's unnecessary.

**Comment by TomPed** [comment]
:+1:.

**Comment by hpinkos** [comment]
@TomPed looks like you missed some alpha variables in `ImageMaterialProperty`
Check the other functions

**Comment by TomPed** [comment]
I think I got the last of the alphas.

**Comment by hpinkos** [comment]
Looks good, thanks @TomPed!

**Pull request merged**